capstone/src/main/java/com/google/sps/servlets/utils/ServletUtil.java

```diff
@@ -1,14 +1,30 @@
 package com.google.sps.servlets;
 
+import com.google.api.client.extensions.appengine.datastore.AppEngineDataStoreFactory;
+import com.google.api.client.extensions.appengine.http.UrlFetchTransport;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.googleapis.auth.oauth2.GoogleClientSecrets;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.calendar.CalendarScopes;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.io.InputStreamReader;
 import java.util.ArrayList;
+import java.util.Collections;
+import javax.servlet.http.HttpServletRequest;
 
 public final class ServletUtil {
+  static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
```

Comment:
Kevin mentioned this in a previous PR - it's better to have one instance of JsonFactory so I moved this to the Constants file for using the Gmail API. Would you be able to add it to Constants.java as well? mainly to resolve any merge conflicts early on